src/TOMLInterface.jl

```diff
@@ -253,39 +276,73 @@ end
         default_param_data,
         save_path,
         save_file,
-        iter=nothing)
+        iter=nothing,
```

Comment:
```suggestion
        iteration
```
consistency with changes.